packages/wallet/src/redux/__tests__/helpers.ts

```diff
@@ -79,51 +79,114 @@ export const expectThisMessage = (state: SideEffectState, messageType: string) =
     expect(item.messagePayload.type).toEqual(messageType);
   });
 };
-export const expectThisMessageAndCommitmentSent = (
-  state: SideEffectState,
-  c: Partial<Commitment>,
-  messageType: string,
-) => {
-  expectSideEffect('messageOutbox', state, item => {
-    expect(item.messagePayload.type).toEqual(messageType);
-    expect(item.messagePayload.signedCommitment.commitment).toMatchObject(c);
-  });
-};
 
-export const expectThisMessageAndTheseCommitmentsSent = (
+type PartialCommitments = Array<{ commitment: Partial<Commitment>; signature?: string }>;
+
+export function transformPartialCommitmentToObjectContainint(sc: {
```

Comment:
Typo here Containint -> Containing (presumably).

Could this function be better named?. Unless I misunderstand, the input type is not a `PartialCommitment` (it is an unnamed type which has a property with type `PartialCommitment`), and it does not return an `Object` (it returns a jest matcher).

Reply:
a Jest matcher is an object!

Reply:
I think it's too wordy to say `transformFromMaybeSignedPartialCommitmentToJestMatcher`. I would be more inclined to say `transformCommitmentToMatcher`. There's no questioning what kind of object it takes -- it's right there in the signature!